llm_unlearn_ucl/unlearn_harm.py

```diff
@@ -477,26 +516,34 @@ def main(args) -> None:
         print(f"Retaining dataset not known! dataset: {args.retaining_dataset}")
         return
 
-    data_sample_artifacts = wandb.Artifact(
-        name="training_batch_raw_data", type="batch_data"
-    )
-    data_sample_artifacts.add_file(
-        normal_sample_path, name=f"normal_{args.samples_count}_samples.json"
-    )
-    data_sample_artifacts.add_file(
-        bad_sample_path, name=f"bad_{args.samples_count}_samples.json"
-    )
-    wandb.log_artifact(data_sample_artifacts)
+    # TODO: TEST THIS
+    if bool(args.wandb_log) and accelerator.is_local_main_process:
```

Comment:
This should prob go [here](https://github.com/Adamliu1/SNLP_GCW/blob/380f2e755a7893f77a80c8adab6d4f461d2a2c1e/llm_unlearn_ucl/data_utils.py#L66)